tools/deployment/mmdet_handler.py

```diff
@@ -1,16 +1,16 @@
 # Copyright (c) OpenMMLab. All rights reserved.
 import base64
```

Comment:
Not critical, but ideally we'd have something like `togal_mmdet_handler.py`, which would overload `inference()` method of `MMdetHandler`. Looks like mmdetection is still actively being worked on, and this way updating from upstream would be easier IMO.

Reply:
That's a good point, I'll add that, thanks!